tasks.py

```diff
@@ -26,6 +27,34 @@ def log_results(out, args):
     #         html_body=h,
     #     )
 
+def snapshot_create(symbol: str):
+    return
+
+
+def snapshots_refresh(limit: int = 5, days: int = 7):
+    symbols = all_nyse_symbols()
+    subset = []
+    for symbol in symbols:
+        # Find or Populate Companies from list
+        company = Company.query.filter_by(symbol=symbol).first()
+        if not company:
+            company = Company.make(symbol)
+            db.session.add(company)
+            db.session.flush()
+        # Check for latest snapshot
+        latest_snapshot = (
+            Snapshot.query.filter_by(company_id=company.id)
+            .order_by(Snapshot.creation_time.desc())
+            .first()
+        )
+        if not latest_snapshot or latest_snapshot.stale(7):
```

Comment:
Seems like we probably want to use the days variable here. ;)
```suggestion
        if not latest_snapshot or latest_snapshot.stale(days):
```

Reply:
Good catch, forgot to include it in this PR, will just do a hotfix to master with it.